src/web_crawler/twir_crawler.rs

```diff
@@ -148,11 +158,23 @@ impl TwirCrawler {
         let issues_and_titles: Vec<TwirLinkElement> = self.get_all_archived_twir_issues().await?;
         let mut found_resources: Vec<TwirLinkElement> = Vec::new();
 
+        // The value it should increment to the bar after each issue check is done
+        let bar_value: f32 = 100.0 / limit as f32;
+
+        let mut current_bar_value = 0.0;
+
+        let progress_bar = get_progress_bar("Checking Issues");
+
         let limit = limit as usize;
 
         for (index, issue) in issues_and_titles.into_iter().enumerate() {
             found_resources.append(&mut self.parse_page(&issue.link, sentence).await?);
 
+            current_bar_value += bar_value;
+
+            let int_bar_value = current_bar_value.round() as u64;
```

Comment:
the name says int_bar_value but it's a u64. I suggest a rename here, could be something like "rounded_bar_value", wdyt?